tests/rkt_metadata_service_test.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+	"testing"
+
+	"github.com/coreos/gexpect"
+	"github.com/coreos/rkt/tests/testutils"
+)
+
+func startMetadataService(t *testing.T, ctx *testutils.RktRunCtx) *gexpect.ExpectSubprocess {
```

Comment:
There is already a function LaunchMDS:

https://github.com/coreos/rkt/blob/master/tests/testutils/ctx.go#L103

If you use it, it will also be automatically stopped on `ctx.Cleanup()`.